installation/resources/installer-cr.yaml.tpl

```diff
@@ -23,3 +23,5 @@ spec:
     - name: "remote-environments"
       namespace: "kyma-integration"
       release: "default-remote-environments"
+    - name: "logging"
+      namespace: "kyma-system"
```

Comment:
please consider adding a new line. Applies to multiple places

Reply:
Hi, where should I add that new line or what new line exactly?